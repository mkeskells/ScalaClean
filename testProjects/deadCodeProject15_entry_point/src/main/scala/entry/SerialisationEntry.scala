package serentry

object Entry extends App {
  new Used()
}

class Used extends BaseUsed {
  private def unused1 = ???
  def unused2 = ???
  private def readResolve = ???
  private def writeReplace = ???
  private def readObject = ???
  private def writeObject = ???
  private def readObjectNoData = ???
}
class Unused extends BaseUnused{
  private def unused1 = ???
  def unused2 = ???
  private def readResolve = ???
  private def writeReplace = ???
  private def readObject = ???
  private def writeObject = ???
  private def readObjectNoData = ???
}
class Unused2 extends BaseUnused{
  private def unused1 = new Unused2
  def unused2 = new Unused2
  private def readResolve = new Unused2
  private def writeReplace = new Unused2
  private def readObject = new Unused2
  private def writeObject = new Unused2
  private def readObjectNoData = new Unused2
}

class BaseUsed extends Serializable
class BaseUnused

