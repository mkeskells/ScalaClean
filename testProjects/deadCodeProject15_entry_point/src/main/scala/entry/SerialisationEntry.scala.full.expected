package serentry

object Entry extends App {
  new Used()
}

class Used extends BaseUsed {
  private def readResolve = ???
  private def writeReplace = ???
  private def readObject = ???
  private def writeObject = ???
  private def readObjectNoData = ???
}

class BaseUsed extends Serializable

